### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by XDRO**
## Describe your changes
Create componenets folder, add Header, Header.tsx, Header.css, Header.stories.tsx. Create App folder, import header there. Import App into main.tsx. Navigate to stories with npm run storybook:node16. Update storybook to 8.2.9
## Issue ticket number and link
https://tripleten-apiary.atlassian.net/browse/EES-9
## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [ ] I have made any necessary updates to Storybook to accompany these changes
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by michaelabudde — APPROVED** [review]
It looks like all of the necessary packages were added and Story book was set up. It also looks like CSS for the Header, App, and Button was added. You did a great job with keeping the code organized and responsive.

File: src/components/Header/Header.css
Comment (RPinkha) [inline src/components/Header/Header.css:1]: The CSS declarations need to be changed to tailwind classes

**Review by RPinkha — COMMENTED** [review]
Tailwind seems to be missing from the header

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <div className="flex justify-between items-center w-full h-[74px] px-[128px] py-[80px] bg-wheat">
+      <Link to="/">
+        {/* run npm run storybook:node16 */}
+        {/* http://localhost:6006/iframe.html?id=components-header--default */}
+        <img src={EES} alt="EES Logo"></img>
+      </Link>
+      <ol className="flex justify-between items-center p-0 m-0 gap-[20px]">
+        <li
+          className="font-text text-[20px] font-medium leading-[18px] text-center cursor-pointer"
```
Comment (RPinkha) [inline src/components/Header/Header.tsx]: Text size needs to be used from the config file.

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          onClick={() => console.log("clicked about us")}
+        >
+          About us
```
Comment (RPinkha) [inline src/components/Header/Header.tsx]: Would it not be better to use an "a" tag inside the "li" for the links?

File: src/components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import { Link } from "react-router-dom";
+export function Header() {
+  return (
+    <div className="flex justify-between items-center w-full h-[74px] px-[128px] py-[80px] bg-wheat">
+      <Link to="/">
+        {/* run npm run storybook:node16 */}
+        {/* http://localhost:6006/iframe.html?id=components-header--default */}
+        <img src={EES} alt="EES Logo"></img>
+      </Link>
+      <ol className="flex justify-between items-center p-0 m-0 gap-[20px]">
```
Comment (RPinkha) [inline src/components/Header/Header.tsx]: Minor nit-pick. Since the items on here are interchangeable, we should use a "ul" tag instead.

**Review by RPinkha — COMMENTED** [review]
Need to fix responsiveness of the header. The logo completely disappears at 600px.
The font isn't rendering as Roboto Flex because the main branch you pulled from is no longer importing Roboto Flex. 
image

File: src/components/Header/Header.tsx (hunk 3 above)
Comment (XDRO) [reply src/components/Header/Header.tsx]: Changed <ol></ol> tag to <ul></ul> tag

File: src/components/Header/Header.tsx (hunk 1 above)
Comment (XDRO) [reply src/components/Header/Header.tsx]: Changed text-[20px] to use tailwind.config.js file text-paragraph-sm

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (XDRO) [reply src/components/Header/Header.tsx]: changed <li></li> tags to <a></a> tags

**Pull request closed without merging**